index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Luis's Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <header>
+      <nav class="flex-nav">
+        <ul>
+          <li class="logo"> <a href="#">My Logo</a></li>
+          <li class="menu-item">Portfolio</li>
+          <li class="menu-item">About</li>
+          <li class="menu-item">Contact</li>
+          <li class="hamburger-menu">☰</li>
+        </ul>
+    </nav>
+  </header>
+
+    <section class="content">
+      <h1 class="title">
+        I'm Luis <br>Glad to see you!
+      </h1>
+      <p class="descr">I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <h2 class="connect"> <a href="#">Let's connect</a></h2>
+      <ul class="socials">
+        <li><a href="https://twitter.com/lujimhe" target="_blank" rel="noopener noreferrer">
+          <img src="https://tinyurl.com/2kabbz4x" alt="Twitter logo" ></a></li>
+        <li><a href="https://www.linkedin.com/in/lujim/" target="_blank" rel="noopener noreferrer">
+          <img src="https://tinyurl.com/yrsm2bzk" alt="Linkedin logo"></a></li>
+        <li><a href="https://medium.com/@lu.jim" target="_blank" rel="noopener noreferrer">
+          <img src="https://tinyurl.com/rrt889rt" alt="Medium logo"></a></li>
+        <li><a href="https://github.com/lu-jim" target="_blank" rel="noopener noreferrer">
+          <img src="https://tinyurl.com/githublogo" alt="Github logo"></a></li>
+        <li><a href="https://angel.co" target="_blank" rel="noopener noreferrer">
+          <img src="https://tinyurl.com/jnb8s2ds" alt="AngelList logo"></a></li>
+      </ul>
+    </section>
```

Comment:
- [ ] The background for this section is still missing, please stick to the design as much as possible. 🚀 
   - I will recommend adding the background image with CSS and the `background-image` property like this:
       ```
          .content {
              background-image: url("./assets/background.png")
          }
       ```   
   - As you can see in this image the Figma template has a background image:
   ![image](https://user-images.githubusercontent.com/60789317/133855505-0ebd2baf-03ee-4bf2-b262-208044e56f67.png)

Reply:
For this one, I am unsure how has this happened twice. On my first attempt, I created a resources folder where I placed all my assets and I referred to them as `./resources/twitter-icon.png" /> `

However, it didn't work and the previous reviewer still saw the logos and the background missing. As I asked for feedback way after the fact, I wasn't able to get a response after I asked for guidance. So for this attempt I tried to host them remotely in a dropbox folder and then use the link for them, but it didn't work either. Any suggestions on what might be happening on my end?